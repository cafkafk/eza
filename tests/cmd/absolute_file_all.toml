bin.name = "eza"
args = "tests/itest/index.svg --absolute"
