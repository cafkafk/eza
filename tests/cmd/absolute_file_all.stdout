[CWD]/tests/itest/index.svg
